wagtail/admin/views/generic/history.py

```diff
@@ -132,7 +135,6 @@ def get_context_data(self, *args, object_list=None, **kwargs):
             "model_name": self.model._meta.verbose_name
         }
         context["header_action_icon"] = "edit"
-        context["index_results_url"] = self.get_history_results_url(self.object)
```

Comment:
In snippets, this view (which extends `BaseListingView`) gets the `index_url_name` and `index_results_url_name` due to their inclusion in the viewset's `get_common_view_kwargs`. The former is still useful to generate the breadcrumbs, but the latter isn't.

Overriding the `index_results_url` context variable would only fix the slim header's search/filter form URL. Other parts e.g. the active filter removal URLs make use of `self.index_results_url` as well. Override the `get_index_results_url` method instead, so this is fixed in both places.

Alternatively, we can also amend e3bf15d2ca8c704d8bc168b7d6228f1c20cb2290 so that the history results URL name is passed as `index_results_url_name` in the viewset's `get_history_view_kwargs` instead.